llvm/docs/AMDGPUUsage.rst

```diff
@@ -1170,6 +1170,23 @@ The AMDGPU backend implements the following LLVM IR intrinsics.
 
   :ref:`llvm.set.fpenv<int_set_fpenv>`             Sets the floating point environment to the specifies state.
 
+  llvm.amdgcn.readfirstlane                        Provides direct access to v_readfirstlane_b32. Returns the value in
+                                                   the lowest active lane of the input operand. Currently 
+                                                   implemented for i16, i32, float, half, bf16, v2i16, v2f16 and types 
```

Comment:
This mixes type naming conventions. Probably should stick to the IR convention (and say `bfloat`, `<2 x i16>`, `<2 x half>`, `<2 x bfloat>`. Also, pointers double, and multiples of the 32-bit vectors should work.